### PR TITLE
Dockerize installation

### PR DESCRIPTION
**Pull request by erikthi**
Set up grafana, influxdb and upd-to-influx using docker-compose. See more information in README.md

Also, fixes a bug with the TWI driver.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,7 @@ LDFLAGS=-lm
 SDKDIR = src/vendor/nrf5-sdk
 LIBDIR = src/vendor/libmira
 
+
```
Comment (Rob1nJ) [inline Makefile]: Should this be here?

File: README.md [hunk 2]
```diff
@@ -1,140 +1,146 @@
...
+## Install docker
+```
+curl -sSL https://get.docker.com | sh
+sudo usermod -aG docker <username>
+```
+
+## Set up influxdb, grafana and udp-to-influx services
+Copy the mira-ruuvitag/RPi directory the the Raspberry Pi.
+```
+scp -r mira-ruuvitag/RPi <username>@<raspberry_pi_ip>:/home/<username>/
```
Comment (Rob1nJ) [inline README.md]: This step should only be done if the setup is done remotely.

File: README.md [hunk 3]
```diff
@@ -1,140 +1,146 @@
...
+## Bulid
+Stand in the top level directory and run
 ```
 make
 ```
+## Flash
+
+First, add a license to the device on address 0x7F000 of length 0x1000, according to MiraOS documentation:
+
+https://docs.lumenrad.io/miraos/2.8.0/description/licensing/licensing_tool.html#signing-a-connected-device
```
Comment (Rob1nJ) [inline README.md]: I think this is the link that should be used: https://docs.lumenrad.io/miraos/2.8.0/description/licensing/licensing_tool.html#signing-a-device-with-remotely-generated-license

File: README.md [hunk 4]
```diff
@@ -1,140 +1,146 @@
...
-
-With this format 11 sensor values can be added before fragmentation occurs.
+   - Payload: 64
 
-Receiving and presenting
------------------------
 
-An example script for receiving sensor updates, and publish to InfluxDB is
-provided as udp\_to\_influx.py
+When commisioning several Ruuvi tags, make sure to set a different name in step 1. All other parameters can remain the same.
```
Comment (Rob1nJ) [inline README.md:110]: Perhaps it's good to mention that the Network PAN ID and Network encryption key need to be the same as what is set in the gateway configuration.

File: Makefile (hunk 1 above)
Comment (erikthi) [reply Makefile]: the newline or nrf5 sdk? Newline should go, nrf5-sdk needs to stay unfortunately.

Comment (Rob1nJ) [reply Makefile]: The newline

File: README.md (hunk 4 above)
Comment (erikthi) [reply README.md:110]: The current README is now using the default values for the gateway. I was thinking about adding a "advanced configuration" section or similar at the end, and leave these details out of the "Quick setup" to keep it as short as possible. Would that make sense?

File: README.md (hunk 2 above)
Comment (erikthi) [reply README.md]: Good point. 

So add one version with steps on the RPi or just assume the user is working on the RPi?

Comment (Rob1nJ) [reply README.md]: I think one can assume that the user is working on the RPi directly.

File: README.md (hunk 4 above)
Comment (Rob1nJ) [reply README.md:110]: Yeah, that would make sense I think.  
But I still think that it's good to emphasize that the keys should be the same on both the tags and the gateway. And possible also state that they should be changed.  
Even in the "Quick setup" section.

Comment (erikthi) [reply README.md:110]: done

File: Makefile (hunk 1 above)
Comment (erikthi) [reply Makefile]: fixed

File: README.md (hunk 2 above)
Comment (erikthi) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (erikthi) [reply README.md]: good catch, fixed

File: README.md (hunk 4 above)
Comment (erikthi) [reply README.md:110]: Have a look now if you like it better.

File: README.md [hunk 5]
```diff
@@ -1,140 +1,123 @@
...
-  - 4 bytes, MSB first, unsigned, sensor Q value
-
-The sensor values is transmitted as a rational value. To get the correct sensor
-value, calculate P/Q.
-
-This method makes it possible for the sensor to provide a proper range and
-resolution for presentation.
-
-With this format 11 sensor values can be added before fragmentation occurs.
+   - Payload: 64
```
Comment (pengi) [inline README.md]: Do it have to be those values? I think those are the ones used in a particular test net, and thus should be replaced to what the user themselves uses for their installation?

Comment (erikthi) [reply README.md]: No. As this is a "Quick start" I don't wanna throw all the options in the face of the user so to speak. I moved the configuration information to a readme under application. I could add a link to that, or just take back what we had before. Up to you.

I thought it was nice to just have some recommended values and an interested user will have to read the documentation anyway to understand what is what. I think in the future these should be the default values, and the node is commissioned by adding a name.

File: README.md (hunk 4 above)
Comment (Rob1nJ) [reply README.md:110]: I think it looks good now!

File: README.md (hunk 5 above)
Comment (erikthi) [reply README.md]: The panid and encryption key is the default values from the gateway

**Review by Rob1nJ — APPROVED** [review]

**Review by pengi — APPROVED** [review]

**Pull request merged**